packages/slider/demo/stories/SliderStory.tsx

```diff
@@ -0,0 +1,210 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { createRef, forwardRef } from 'react';
+import { Story } from '@storybook/react';
+import {
+  ISliderContainerProps,
+  IUseSliderProps,
+  IUseSliderReturnValue,
+  SliderContainer,
+  useSlider
+} from '@zendeskgarden/container-slider';
+import classNames from 'classnames';
+
+interface IComponentProps extends IUseSliderReturnValue {
+  max: IUseSliderProps['max'];
+  min: IUseSliderProps['min'];
+  minThumbRef: IUseSliderProps<Element, HTMLDivElement>['minThumbRef'];
+  maxThumbRef: IUseSliderProps<Element, HTMLDivElement>['minThumbRef'];
+  disabled: IUseSliderProps['disabled'];
+  rtl: IUseSliderProps['rtl'];
+}
```

Comment:
Is there a particular reason this interface lives here, instead of in `types.ts`? (Is the story the only place we use it?)

Reply:
Yes, this is local to the story and consistent with the demo typing for all other `react-containers` stories.